pkg/logging/logger.go

```diff
@@ -35,9 +47,20 @@ func init() {
 	}()
 
 	logger = zapLogger.Sugar()
+
+	if configErr != nil {
+		logger.Info(configErr)
+	}
+	logger.Infof("Logging at %s level", zapLogger.Level())
 }
 
 func WithLogger(ctx context.Context) context.Context {
+	if logger == nil {
+		// defaults to Debug if InitLogger has not been called.
+		// all cli commands should call InitLogger, so this should mostly be for unit tests
+		InitLogger("Debug")
```

Comment:
Zapcore provides `ParseLevel` which handles parsing a string to the zapcore level type, so I don't think it makes sense to add another enum and do the converting of a string to it ourselves. I added consts for the strings, which I think is OK for now (they are only referenced twice) ?

Reply:
I think the way i'm thinking about it is that the enum acts as a documentation of what values "InitLogger" can take. I'd say maybe if not an enum then some documentation around the InitLogger function to provide what values it can receive. (since it is an abstraction on the zapcore logger, which may  change in the future)